src/util.cpp

```diff
@@ -1436,6 +1436,27 @@ bool CLocale::IsCountryCodeSupported ( unsigned short iCountryCode )
 #endif
 }
 
+QLocale::Country CLocale::GetCountryCodeByTwoLetterCode ( QString sTwoLetterCode )
+{
+    QList<QLocale> vLocaleList = QLocale::matchingLocales ( QLocale::AnyLanguage, QLocale::AnyScript, QLocale::AnyCountry );
+    QStringList    vstrLocParts;
+
+    // Qt < 6.2 does not support lookups from two-letter iso codes to
```

Comment:
We have Qt6 code that does `#if` on the version, don't we?  Why not here?

Reply:
Yes, we do. At most places it's about different *required* logic per Qt version though (and we don't have minor version dependent `#if`s for Qt6, IIRC).

So, the downside is that we have different implementations for Qt5 and Qt6 although we could live with a single one (less complexity, risk for differences between that two implementations).
The upside is that we would have the "better" code in place right now, therefore making it easier to drop the legacy path in X years.

I can add a 6.2+ code path via `QLocale::codeToTerritory()` as part of this PR.
Would you prefer that?

(Additionally, we could even have a 6.1 code path via `QLocale::codeToCountry`, which was introduced and deprecated right away due to the renaming to `QLocale::codeToTerritory` in 6.2+. I guess this step would really not be worth it as we don't use 6.1 anywhere in official releases. It would therefore release zero testing.)

Reply:
I think there was a place with commented Qt5 code and a Qt4 implementation with the country codes. We could do the same here.

Reply:
> I think there was a place with commented Qt5 code and a Qt4 implementation with the country codes.

Yep.

> We could do the same here.

I guess we would drop the whole helper method `GetCountryCodeByTwoLetterCode` in favor of the native Qt6.2+ function. So this would probably boil down to just updating the comment with the proper Qt native method name?

Reply:
So the two paths would either be "call the native method" or "call the compatibiltity method", depending on the Qt version?

Reply:
> So the two paths would either be "call the native method" or "call the compatibiltity method", depending on the Qt version?

Exactly.

Reply:
I've added that now.
I've also added the suggested logging in a seperate commit (initially proposed in https://github.com/jamulussoftware/jamulus/pull/2829#discussion_r965316253)